lib/discordrb/cache.rb

```diff
@@ -147,10 +147,14 @@ def ensure_user(data)
 
     # Ensures a given server object is cached and if not, cache it from the given data hash.
     # @param data [Hash] A data hash representing a server.
+    # @param force_cache [true, false] Whether the object in cache should be updated with the given
```

Comment:
The `Boolean` placeholder type is accepted yard convention so we could probably use that here.